### PR TITLE
Adjusted lookup methods in api definition.

### PR DESCRIPTION
**Pull request by JoepKillaars**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a logo.

### CODE REVIEW

File: certified-connectors/AXtension Content Gate/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/AXtension Content Gate/apiDefinition.swagger.json:331]: @JoepKillaars Why there is a long space between `The friendly name of the` and `file type of the content entity." ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@JoepKillaars I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by sriyen-msft** [comment]
@JoepKillaars I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by sriyen-msft** [comment]
@JoepKillaars I am closing this PR with out approving and merging it since there is no response from your side and no updates have been pushed to the given feedback. You can reopen the PR whenever you are ready.

**Comment by JoepKillaars** [comment]
Hi @sriyen-msft,

There is another update coming, and I am waiting for approval to get it released. I will update this PR if the new changes are approved for release.

Thanks !

**Pull request closed without merging**
